src/renderer/components/ft-list-video/ft-list-video.js

```diff
@@ -225,6 +233,9 @@ export default defineComponent({
     },
 
     thumbnail: function () {
+      if (this.thumbnailPreference === 'hidden') {
+        return require('../../assets/img/thumbnail_placeholder.svg')
+      }
```

Comment:
Nitpick but would prefer this being moved inside the switch case that is only a few lines below it.

Reply:
If we made it a switch case we would have to run another check in the else block above which is less clean IMO.

Reply:
I guess you mean `baseUrl` is not required when `hidden`?

Reply:
My concern is actually related to the implementation of this change in `playlist-info.js` not `ft-list-video.js` (my bad). Making this change here in `ft-list-video.js` is actually not a problem. I've left a comment in the relevant thread.